src/features/projects/releases/releaseRepository.ts

```diff
@@ -13,6 +13,12 @@ export class ReleaseRepository {
     }
 
     async create(command: CreateReleaseCommandV1): Promise<CreateReleaseResponseV1> {
+        const capabilityError = await checkForCapability(this.client, "CreateReleaseCommandV1");

```

Comment:
Is there a way we can link between the command that we are creating and the capability string? Perhaps the capability id could go in the Command object itself.
Just a thought

Reply:
What I'd love is for TS to support some equivalent to `typeof(CreateReleaseCommandV1).Name` that we have in C#, but it doesn't have such support.

Reply:
yeah, that was the sort of thing I had in mind